docs/UserGuide.md

```diff
@@ -105,9 +109,10 @@ Examples:
 
 ### Editing a student: `edit`
 
-Edit the specific student detail from TuteeTally using the ID.
-
-Parameters: INDEX (must be a positive integer) [-nameNAME] [-phonePHONE] [-emailEMAIL] [-addressADDRESS] [-subjectSUBJECT] [t/TAG]... 
```

Comment:
Hmm, did you edit the error msg for invalid edit command to remove the subject yet ? <or is this not allowed, im not sure :/ > 

Regarding issue #177 

![image](https://github.com/AY2324S2-CS2103T-F10-2/tp/assets/122339963/d1f36f94-69b0-4411-8b59-a639524a9d08)

Reply:
I will amend this together with another command error prompt issue! See issue #163